netty/src/main/java/io/grpc/netty/NettyServer.java

```diff
@@ -113,7 +115,7 @@
       long maxConnectionIdleInNanos,
       long maxConnectionAgeInNanos, long maxConnectionAgeGraceInNanos,
       boolean permitKeepAliveWithoutCalls, long permitKeepAliveTimeInNanos,
-      InternalChannelz channelz) {
+      Attributes eagAttributes, InternalChannelz channelz) {
```

Comment:
Any reason why the new param wasn't added to the end?

Reply:
I actually went back-and-forth on it. Here it mirrors the NettyServerTransport and is more natural looking in NettyServerBuilder as channelz is not just a plain field in the builder. It doesn't matter much.